src/SumOfDigits.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Created by Yura on 01.05.2019.
+ */
+
+import java.util.Scanner;
+
+public class SumOfDigits {
+
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+
+        int n = in.nextInt();
+        int sum = 0;
+
+        while(n != 0){
+
```

Comment:
get rid of all redundant empty lines, try to separate this way logical units only